br/pkg/lightning/backend/local/localhelper.go

```diff
@@ -478,13 +481,9 @@ func (local *local) checkScatterRegionFinishedOrReScatter(ctx context.Context, r
 		if respErr.GetType() == pdpb.ErrorType_REGION_NOT_FOUND {
 			return true, nil
 		}
-		// don't return error if region replicate not complete
```

Comment:
We don't need to handle this error now, do we?

Reply:
It is moved to retriable errors.

Reply:
oh, it's caught by the caller.